### DISCUSSION TITLE
Fu Bao's Birthday Special Update Before 2.22.2 Major

### DISCUSSION THREAD

**Discussion by Outfit8TSB** [EssentialsX/Essentials#5873] [Ideas and Feature Suggestions]
🐼HAPPY 4TH BIRTHDAY FUBAO! :) 🎂 
Expected Release Date: 2024-07-20, When FuBao's 4th birthday!
Functions: add a easter egg command `/baofamily`, which Spawn pandas named: "LeBao", "AiBao", "FuBao", "HuiBao", "RuiBao" 
when the command triggered on July 7, July 13, July 20, July 28. July  Jul 7:Huibao and RuiBao, Jul 13: AiBao, Jul 20: FuBao, Jul 28: LeBao

**Comment by Outfit8TSB** [comment]
2.22.2 Major is a secret, you will never know about this pull request.

**Comment by mdcfe** [comment]
Please refrain from posting spam on the issue tracker and feature request boards.
